apps/wizard/templating/snapshot.py

```diff
@@ -806,4 +806,6 @@ def run_snap_step() -> None:
     # Run step
     with st.spinner(f"Running snapshot step... {command_str}"):
         subprocess.call(args=commands)
-    st.write("Snapshot uploaded!")
+    st.write(
```

Comment:
Could you not catch the response from the subprocess call and create the message accordingly? If not, this solution is fine.